src/views/pages/Inbox/index.js

```diff
@@ -41,7 +41,10 @@ function Inbox() {
       setMessageList(d && d.length > 0 ? d : null);
     });
 
-    messageService.checkMessages(defaultSender);
+    console.log(`Getting messages for ${defaultSender}`);
+    if (!!defaultSender) {
+      messageService.checkMessages(defaultSender);
+    }
```

Comment:
this is doing the same as the pipe-filter was doing, except now with an if statement